pytest_rerunfailures.py

```diff
@@ -30,6 +31,14 @@ def pytest_addoption(parser):
     group = parser.getgroup(
         "rerunfailures",
         "re-run failing tests to eliminate flaky failures")
+    group._addoption(
+        '--only-rerun',
+        action='store',
+        dest='only_rerun',
+        type=str,
+        default=None,
+        help='Optional comma separated list. If passed, only rerun errors matching the regexes provided.'
```

Comment:
I was wondering if there is the ability to use this option multiple times, so we get a list of regexes from the parser. This would allow commas in the regex and the shell eliminates not necessary whitespace for us.

Reply:
Let me check tonight ( I’m in EST )

Reply:
setting action=append for `group._addoption` does that. Thank you for bringing this up, I updated the PR accordingly